presto-main/src/main/java/com/facebook/presto/block/BlockEncodingManager.java

```diff
@@ -60,7 +59,7 @@ public BlockEncodingManager(TypeManager typeManager, Set<BlockEncodingFactory<?>
     {
         // This function should be called from Guice and tests only
 
-        this.typeManager = requireNonNull(typeManager, "typeManager is null");
+        requireNonNull(typeManager, "typeManager is null");
```

Comment:
Can be removed altogether, since `typeManager` is just passed through. Let `MapBlockEncodingFactory` and `SingleMapBlockEncodingFactory` validate that